dbscripts/xml/install.xml

```diff
@@ -41,6 +41,7 @@
 	<migration class="PKP\migration\install\InstitutionsMigration" />
 	<migration class="PKP\migration\install\AffiliationsMigration" />
 	<migration class="PKP\migration\install\RorsMigration" />
+	<migration class="PKP\migration\upgrade\v3_5_0\I10359_DateConsideredToReviewAssignments" />
```

Comment:
We don't need this here. Script for review assignments installation is here: https://github.com/pkp/pkp-lib/blob/ba0213f2553113b6635e901ef8ebd3dce245d0ba/classes/migration/install/ReviewsMigration.php#L48